apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx

```diff
@@ -316,21 +290,21 @@ function TryEnclave({
     <View style={styles.onboardingScreen}>
       <View style={styles.createAccountPage}>
         <TextH1>
-          <Octicons name={askWeak ? "unlock" : "lock"} size={40} />
+          <Octicons name={askToSetPin ? "unlock" : "lock"} size={40} />
         </TextH1>
         <Spacer h={32} />
         <View style={ss.container.padH16}>
           <TextCenter>
-            {!askWeak && (
+            {!askToSetPin && (
               <TextBody>
                 Create a cryptographic key on-device that will authorize your
                 account. This makes your money yours alone.
               </TextBody>
             )}
-            {askWeak && (
+            {askToSetPin && (
               <TextBody>
-                Failed to create hardware authenticated key. This can happen on
-                older Android devices. Use regular key?
+                Your device does not appear to have a secure lock screen (PIN or
+                biometrics) set up. Please set one up and try again.
```

Comment:
given this will also show up the user cancels (and potentially other device issues), maybe

```
const noun = platform === 'android' ? 'lock screen' ? 'passcode';
```
```
Authentication failed. Does your phone have a ${noun}? You'll need one to secure your Daimo account.```

Reply:
This screen is android only. Will update copy with noun = `lock screen (PIN or biometrics)`.